csharp/rocketmq-client-csharp/Client.cs

```diff
@@ -518,11 +508,16 @@ internal virtual void OnSettingsReceived(rmq::Settings settings)
 
         protected readonly IClientManager Manager;
 
-        protected readonly HashSet<string> _topicsOfInterest = new HashSet<string>();
+        protected readonly ConcurrentDictionary<string, bool> _topicsOfInterest = new ();
```

Comment:
Why do we need a map here?

Reply:
With this change, we will support dynamically subscribe/unsubscribe to multiple topics, maybe from different threads. C# does not have a ConcurrentSet, thus a ConcurrentDictionary to emulate one.

Reply:
OK